receiver/sqlserverreceiver/README.md

```diff
@@ -36,6 +38,14 @@ Windows-specific options:
 - `computer_name` (optional): The computer name identifies the SQL Server name or IP address of the computer being monitored.
   If specified, `instance_name` is also required to be defined. This option is ignored in non-Windows environments.
 
+Query-level collection specific options (only useful when query-level collection are enabled):
```

Comment:
I found that `query-level collection` and `Top query collection` are used interchangeably in the document, which is quite ambiguous.

Could you unify the term?

And, if these options only work for `top query collection`, could we put these options inside a sub-option? For instance

```
top_query_collection:
  enabled: true
  lookback_time:
  max_query_count:
  top_active_query_count:
```

Reply:
updated

Reply:
The updated config is

```yaml
logs:
  top_query_collection:
    enabled: true
    lookback_time: 60
    max_query_sample_count: 1000
    top_query_count: 200
```

Is there a reason to use `logs` in the top level of config?

Could we use something like

```
top_query_collection:
  enabled: true
  lookback_time: 60
  max_query_sample_count: 1000
  top_query_count: 200
```

Reply:
updated